src/api/useTasks.ts

```diff
@@ -27,7 +27,7 @@ const fetcher = async (
 
 const useTasks = (id: any): UseTasksObjectDataResponse => {
   const { data, error } = useSWR<GetTasksByFetcher>(ENDPOINT.tasksList.replace('{:id}', id), fetcher);
-
+  
```

Comment:
Adding whitespace as the only change in a file? :thinking: Why not configure your IDE to trim trailing whitespace?